app/controllers/api/v1/gc_organisations_controller.rb

```diff
@@ -42,7 +42,7 @@ def find_record_and_render_json(serializer)
       if params['ids'].present?
         records = @organisations.where(id: params['ids']).page(params["page"]).per(params["per_page"] || DEFAULT_SEARCH_COUNT)
       else
-        records = @organisations.search(params["searchText"]).page(params["page"]).per(params["per_page"] || DEFAULT_SEARCH_COUNT)
+        records = Organisation.search(params["searchText"]).page(params["page"]).per(params["per_page"] || DEFAULT_SEARCH_COUNT)
```

Comment:
```suggestion
        records = @organisations.search(params["searchText"]).page(params["page"]).per(params["per_page"] || DEFAULT_SEARCH_COUNT)
```

Reply:
@steveyken Good point, however the `search` field isn't a scope. Since I had to do a custom `select` with the similarities within it. Any suggestions ?
 can we "update" the select of an existing relation ?

Reply:
@patrixr I think you can scope it in ClassMethods
```
module FuzzySearch
  module ClassMethods
    def search
    end
  end
end
```


I think `@organisations.search` will work. Check `package_filtering.rb` and `packages_controller:151`